TRANSLATIONS.md

```diff
@@ -8,6 +8,12 @@ There are [**translations**](https://github.com/microsoft/IoT-For-Beginners/tree
 
 Translated lessons should follow this naming convention:
 
+When translating the initial Readme , link the other translations to your translated readme and link yours to the main English Readme by using Shields as shown:
+
+<-code in markdown->
+```
```

Comment:
Can you mark this code block as markdown so it gets correctly syntax highlighted please. Should be `markdown` after the opening backticks.

Reply:
> Can you mark this code block as markdown so it gets correctly syntax highlighted please. Should be `markdown` after the opening backticks.

Do you mean that after the opening backticks I have to use '>' for the code ?

Reply:
Check out the markdown syntax highlighting guide:

https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

you need to put the code language on the same line as the opening back ticks, immediately after them. So for this, you need:

````
```markdown
[![Bengali](https://img.shields.io/badge/-Bengali-blue)](translations/README.bn.md)
```
````

Reply:
> Can you mark this code block as markdown so it gets correctly syntax highlighted please. Should be `markdown` after the opening backticks.

Hi, I apologise if I'm a bit behind in this. I'm a bit confused as in the preview this is what I see.

Reply:
> > Can you mark this code block as markdown so it gets correctly syntax highlighted please. Should be `markdown` after the opening backticks.
> 
> Hi, I apologise if I'm a bit behind in this. I'm a bit confused as in the preview this is what I see.


<img width="727" alt="Screen Shot 2021-07-23 at 11 40 18 PM" src="https://user-images.githubusercontent.com/87066683/126823718-120eb8e2-cadf-41ec-b27f-b771c0b4a879.png">

Reply:
I've edited the file to show what I mean